src/Game.h

```diff
@@ -99,6 +99,17 @@ class Game {
 	void SetTimeAccel(TimeAccel t);
 	void RequestTimeAccel(TimeAccel t, bool force = false);
 
+    /**
+     * Requests an increase in time acceleration
+     * @param force if set to false the system can reject the request under certain conditions
+     */
+    void RequestTimeAccelInc(bool force = false);
```

Comment:
@impaktor my comments are not meant to be C-style to be true, they are Javadoc style instead. I am used to this system from my Java experience and I tend to use it in any context because I find it very explicative. In addition, there are several tools (e.g. Doxygen) which would allow us to easily generate an online documentation browser from C++ source code that is annotated this way.

That said I have of course no problems to switch to a 1-line C++ style commenting system if you prefer.

Will modify the comments and amend the commit as soon as I can.